HinglishULMFiT_AWD_QRNN.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>What was the sp model trained on? Can we add that as a comment here please?</p><p> </p><p>We should add this somewhere in the repo README and paper as well maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Used <code>data/interim/hinglish_unsup_high_confidence.json</code> and <code>data/interim/hinglish_unsup_less_confidence.json</code>.   The training notebook is <a href="https://github.com/NirantK/Hinglish/blob/sentencepiece/SentencePieceTokenizer.ipynb" target="_blank">here</a></p><p>Some details- </p><ul><li>Vocab - 8k </li><li>Training data size - 540k tweets</li><li>model_type - bpe</li></ul>

Reply:
Let's move these details to Markdown for easier accessibility and re-use for later experiments? Create a table/leaderboard which notes these somewhere? We are basically benchmarking stuff and if we don't document our config - we waste effort by either losing experiments or repeating them.

Reply:
Yeppp, should I add this in another PR that we will create for documentation?